<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDiagramPageSetupDialog</class>
 <widget class="QDialog" name="QDiagramPageSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Page Setup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabDiagramSize">
      <attribute name="title">
       <string>Size &amp;&amp; Orientation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Paper Size</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QRadioButton" name="autoExpandRradioButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Auto Expand</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="predefinedSizeRadioButton">
            <property name="text">
             <string>Predefined Size</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="paperSizeComboBox"/>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="userDefinedSizeRadioButton">
            <property name="text">
             <string>User Defined Size</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSpinBox" name="pageWidthSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="suffix">
               <string> mm</string>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="pageHeightSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="suffix">
               <string> mm</string>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Paper Orientation</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="portraitRadioButton">
            <property name="text">
             <string>Portrait</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="landscapeRadioButton">
            <property name="text">
             <string>Landscape</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabProperties">
      <attribute name="title">
       <string>Properties</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nameLineEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>QDiagramPageSetupDialog</receiver>
   <slot>buttonBoxClicked(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userDefinedSizeRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>QDiagramPageSetupDialog</receiver>
   <slot>userDefinedSizeRadioButtonToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>portraitRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>QDiagramPageSetupDialog</receiver>
   <slot>portraitRadioButtonToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>paperSizeComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>QDiagramPageSetupDialog</receiver>
   <slot>paperSizeComboBoxActivated(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>predefinedSizeRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>QDiagramPageSetupDialog</receiver>
   <slot>predefinedSizeRadioButtonToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonBoxClicked(QAbstractButton*)</slot>
  <slot>userDefinedSizeRadioButtonToggled(bool)</slot>
  <slot>portraitRadioButtonToggled(bool)</slot>
  <slot>paperSizeComboBoxActivated(int)</slot>
  <slot>predefinedSizeRadioButtonToggled(bool)</slot>
 </slots>
</ui>
